plugin.video.infowars/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon id="plugin.video.infowars"
        name="Infowars"
-       version="2.0.3"
+       version="2.0.4"
        provider-name="Thomas Meadows | Spinalcracker">

```

Comment:
can you make it
provider-name="Thomas Meadows, Spinalcracker"
s it show properly in Kodi and our addon website?
This can be done on next update.